### DISCUSSION TITLE
How to use speechbrain to run Speaker Diarization ?

### DISCUSSION THREAD

**Discussion by amitli1** [speechbrain/speechbrain#1916] [General]
Is there a simple example that shows how to use `speechbrain` in order to run `Speaker Diarization` on test wav file ?

**Comment by asumagic** [comment]
I don't think there is a ready to use interface for this yet. PR #1727 introduces an interface that seems to use existing VAD+speaker embedding models to perform diarization. You may be able to pull those changes if you feel adventurous.

Alternatively, [`pyannote/speaker-diarization`](https://huggingface.co/pyannote/speaker-diarization) should do the trick.
